File: official/docs/php/customs_items/CreateCustomsItems.php
<?php

require_once("../lib/easypost.php");
\EasyPost\EasyPost::setApiKey("EASYPOST_API_KEY");

$customs_item = \EasyPost\CustomsItem::create(array(
    "description" => 'T-shirt',
    "quantity" => 1,
    "weight" => 5,
    "value" => 10,
    "hs_tariff_number" => '123456',
    "origin_country" => 'US'
  ));

echo $customs_item;

File: official/docs/go/customs_items/create_customs_items.go
package example

import (
    "os"

    "github.com/EasyPost/easypost-go/v2"
)

func main() {
	apiKey := os.Getenv("EASYPOST_API_KEY")
    client := easypost.New(apiKey)

	customsItem, err := client.CreateCustomsItem(
        &easypost.CustomsItem{
            Description:   "T-shirts",
            Quantity:      1,
            Value:         10.00,
            Weight:        5,
            OriginCountry: "US",
        },
    )

	fmt.Println(customsItem)
}
